sky/provision/gcp/instance.py

```
@@ -145,11 +150,18 @@ def terminate_instances(
     handler_to_instances = _filter_instances(handlers, project_id, zone,
                                              label_filters, lambda _: None)
     operations = collections.defaultdict(list)
+    errs = []
     for handler, instances in handler_to_instances.items():
         for instance in instances:
-            operations[handler].append(
-                handler.terminate(project_id, zone, instance))
+            try:
+                operations[handler].append(
+                    handler.terminate(project_id, zone, instance))
+            except errors.HttpError as e:
+                if _RESOURCE_NOT_FOUND_PATTERN.search(e.reason) is None:
```

When we ignore this error, probably should add a warning log?